backend/test/Controllers/Admin/UserControllerTest.cs

```diff
@@ -59,20 +59,20 @@ public UserControllerTest()
             _expectedAccessRequest.Agencies.Add(new AccessRequestAgency()
             {
                 AgencyId = AGENCY_ID,
-                Agency = new Entity.Agency()
-                {
-                    Id = AGENCY_ID
-                },
-                AccessRequestId = ACCCESS_REQUEST_ID
+                    Agency = new Entity.Agency()
```

Comment:
haha, yep. this looks familiar.

Reply:
Wish vscode didn't format this way though...